src/config/env.ts

```diff
@@ -44,80 +43,81 @@ const getProxyFromEnvironment = (): string | undefined => {
 	return proxyHost && proxyPort ? `http://${proxyHost}:${proxyPort}` : undefined;
 };
 
-export const envVars: EnvVars = {
+process.env = {
 	...process.env,
 	MICROS_ENV: EnvironmentEnum[process.env.MICROS_ENV || EnvironmentEnum.development],
-	MICROS_SERVICE_VERSION: process.env.MICROS_SERVICE_VERSION,
 	MICROS_GROUP: process.env.MICROS_GROUP || "",
 	NODE_ENV: nodeEnv,
-	SENTRY_DSN: process.env.SENTRY_DSN,
-	JIRA_LINK_TRACKING_ID: process.env.JIRA_LINK_TRACKING_ID,
 	PROXY: getProxyFromEnvironment(),
 	GITHUB_REPO_URL: "https://github.com/atlassian/github-for-jira"
-} as EnvVars;
+};
 
-// TODO: Make envvars dynamic
 // Check to see if all required environment variables are set
-const missingVars = requiredEnvVars.filter(key => envVars[key] === undefined);
+const missingVars = requiredEnvVars.filter(key => process.env[key] === undefined);
 if (missingVars.length) {
 	throw new Error(`Missing required Environment Variables: ${missingVars.join(", ")}`);
 }
 
-export interface EnvVars {
-	NODE_ENV: EnvironmentEnum,
-	MICROS_ENV: EnvironmentEnum;
-	MICROS_SERVICE_VERSION?: string;
-	MICROS_GROUP: string;
-	SQS_BACKFILL_QUEUE_URL: string;
-	SQS_BACKFILL_QUEUE_REGION: string;
-	SQS_PUSH_QUEUE_URL: string;
-	SQS_PUSH_QUEUE_REGION: string;
-	SQS_DEPLOYMENT_QUEUE_URL: string;
-	SQS_DEPLOYMENT_QUEUE_REGION: string;
-	SQS_BRANCH_QUEUE_URL: string;
-	SQS_BRANCH_QUEUE_REGION: string;
+// Adding session information to express Request type
+declare global {
+	// eslint-disable-next-line @typescript-eslint/no-namespace
+	namespace NodeJS {
+		interface ProcessEnv {
+			NODE_ENV: EnvironmentEnum,
```

Comment:
Not sure why this, but personally I prefer keep the `git diff` to minimum as much as possible.